know/rdf/parser/wikidata_parser.py

```diff
@@ -0,0 +1,23 @@
+
+from .abstract_parser import AbstractParser
+
+class WikidataParser(AbstractParser):
+    """parses wikidata sources"""
+
+    def __init__(self, entity_id):
+        self.entity_id = entity_id
```

Comment:
```suggestion
    def __init__(self, uri: string):
        self.uri = uri
```
it's going to be the raw URI